Upstream says:
# XXX: We need to pin some dependencies to keep our MSRV at 1.34.  If
# you are packaging Sequoia, feel free to drop these pins if your
# compiler is newer.

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -26,15 +26,12 @@
 optional = true
 
 [dependencies.flate2]
-version = ">= 1.0.1, < 1.0.16"
+version = "1.0.1"
 optional = true
 
 [dependencies.libc]
 version = "0.2.33"
 
-[dependencies.miniz_oxide]
-version = "= 0.3.7"
-
 [features]
 compression = ["compression-deflate", "compression-bzip2"]
 compression-bzip2 = ["bzip2"]
